cmd/minishift/cmd/config/view.go

```diff
@@ -19,13 +19,14 @@ package config
 import (
 	"bytes"
 	"fmt"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
```

Comment:
Please, do not move this out of place...
We separate on golang provided/framework imports and 3rd party+our own...

Reply:
Updated